pkg/config/config.go

```diff
@@ -390,3 +391,14 @@ func isConfigProvided(cliArgName string, envVarName string) bool {
 	})
 	return cliArgProvided
 }
+
+func getRegionFromQueueURL(queueURL string) string {
+	for _, partition := range endpoints.DefaultPartitions() {
+		for regionID := range partition.Regions() {
+			if strings.Contains(queueURL, regionID) {
+				return regionID
+			}
+		}
+	}
+	return ""
```

Comment:
will this lead to an error?

Reply:
if they do end up changing the queue urls or something, maybe we should at least log here

Reply:
added a debug log

Reply:
oh and to answer the first question, yes it will lead to an aws sdk error that the region is undefined